src/qopenglwebpage.cpp

```diff
@@ -81,6 +83,7 @@ void QOpenGLWebPage::createView()
         d->mContext->GetApp()->SetIsAccelerated(true);
         d->mView = d->mContext->GetApp()->CreateView(mParentID, mPrivateMode);
         d->mView->SetListener(d);
+        d->mView->SetDPI(QGuiApplication::primaryScreen()->logicalDotsPerInch());
```

Comment:
I still think that this should be ```physicalDotsPerInch``` and "apz.touch_start_tolerance" pref adjusted accordingly.

Thinking also this one:

```
AsyncPanZoomController::GetTouchStartTolerance()
{
  return (gfxPrefs::APZTouchStartTolerance() * APZCTreeManager::GetDPI());
}
```